src/components/tutorial-card/helpers/build-aria-label.ts

```diff
@@ -12,21 +12,25 @@ const PRODUCT_LABEL_MAP: Record<ProductOption, string> = {
 	waypoint: 'Waypoint',
 }
 
+export function getSpeakableDuration(duration: TutorialCardProps['duration']) {
+	const speakableDuration = duration
+		.replace('hr', ' hour')
+		.replace('min', ' minute')
+	return `${speakableDuration} tutorial.`
+}
+
 export function buildAriaLabel({
 	heading,
-	duration,
 	productsUsed,
 	hasVideo,
 	hasInteractiveLab,
+	eyebrowSlotAriaLabel,
```

Comment:
I needed a way to have the `aria-label` reflect progress. With our `eyebrowSlot` flexibility on the side of things we can render visibly, it felt right to reflect that flexibility in the `aria-label`. I've gone with `eyebrowSlotAriaLabel` to reflect this intent.

Note that we continue to default this to speakable duration in cases where `eyebrowSlot` renders `duration` - see `tutorial-card` for that default logic.